Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
+            case print
+            case nslog
+        }
+
+        public var minLevel = Log.Level.error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+        public var output = ConsoleOutput.print
```

Comment:
@portellaa the "rule" I mentioned was thinking where to use or not type inference.
In your last example, if we had more 3 or 4 properties, what would you do (meaning, preference)?
For me, I don't have any problem having mixed situations, but I think this is now a matter of personal taste. 😀